packages/tocco-ui-showcase/src/toccoUtilExamples/notifier/example.js

```diff
@@ -5,40 +5,52 @@ import {appFactory, notifier} from 'tocco-util'
 import {Button, ButtonGroup, Typography} from 'tocco-ui'
 // real-import:import {appFactory, notifier} from 'tocco-util'
 
+/* start example */
 const longText = `Lorem ipsum dolor sit amet, at sed inermis intellegam scriptorem, usu facete apeirian ad.
 Sit et meliore intellegam. Mel cu maluisset philosophia, pri et habeo oportere. Vis in purto verear luptatum, has
 ne graecis qualisque. Mei ei placerat incorrupte adversarium, eum rebum nonumy ut.`
 
-/* start example */
+const TitleComponent = <Typography.H4>Main title <Typography.Small>additional byline</Typography.Small></Typography.H4>
```

Comment:
This is not a component.  Hence it should not be uppercase and should be named otherwise.